css/normalize.css

```diff
@@ -0,0 +1,424 @@
+/*! normalize.css v4.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Nothing wrong with a good normalize, but be aware, if things start getting weird and you can't trace it down, try commenting out the tag including the normalize css, it may be doing weird unexpected things.